src/librustc/diagnostics.rs

```diff
@@ -284,6 +284,30 @@ This error indicates that an attempt was made to divide by zero (or take the
 remainder of a zero divisor) in a static or constant expression.
 "##,
 
+E0038: r##"
+This trait is not object safe, but the coercion of an implementation to a trait
```

Comment:
This is super nitpicky, but technically there are two ways to get a trait object value: casting and coercion. Instead of saying "coercion", it might be better to be vague here, something like "It is not allowed to create a trait object value for a trait that is not object safe"